### ISSUE TITLE
Styling fixes for the table component

### ISSUE THREAD

**Issue by IrakliJani** [lightdash/lightdash#4558]
### Description

- [ ] frozen columns are divided by a shadow (it does not match our designs). We should elevate with darker BG colour and add a bit of a lighter shadow.
- [ ] borders are missing in headers
- [ ] borders are off with total.

![image](https://user-images.githubusercontent.com/962095/220599026-446b3163-436f-428b-82e9-9ed52ad36162.png)

**Comment by muhammadjufry** [comment]
Hi!, @IrakliJani Can i work on this issue? Thanks

**Comment by IrakliJani** [comment]
hey @muhammadjufry yep, go ahead.

**Comment by muhammadjufry** [comment]
Sorry for being late answer, Ok thanks

**Comment by muhammadjufry** [comment]
Hi @IrakliJani I cannot find the charts table component file
When i tried to figure out i found this file which i think maybe the table chart will inside this file: packages\frontend\src\components\common\CollabsableCard.tsx

But when i try to find the table chart code i only see this code `{children}` on line 101 but i cannot get the children code
So can you tell me please where the code of the table chart component?

Many thanks

**Comment by IrakliJani** [comment]
hi @muhammadjufry check out LightdashVisualization component which renders SimpleTable

**Comment by muhammadjufry** [comment]
yes i just check it out but i mean where is the code something like this: `<table><something>`
thank you

**Comment by muhammadjufry** [comment]
Sorry for the delay, i just got stuck on running the app but i have fixed it
